### DISCUSSION TITLE
BadMethodCallException Call to undefined method App\User::isRecoveryTeamMember()

### DISCUSSION THREAD

**Discussion by shifa7474** [spatie/laravel-permission#2119] [General]
Illuminate\Database\Eloquent\Model::throwBadMethodCallException
vendor/laravel/framework/src/Illuminate/Support/Traits/ForwardsCalls.php:50

            $pattern = '~^Call to undefined method (?P<class>[^:]+)::(?P<method>[^\(]+)\(\)$~';

 

            if (! preg_match($pattern, $e->getMessage(), $matches)) {

                throw $e;

            }

 

            if ($matches['class'] != get_class($object) ||

                $matches['method'] != $method) {

                throw $e;

            }

 

            static::throwBadMethodCallException($method);

        }

    }

 

    /**

     * Throw a bad method call exception for the given method.

     *

     * @param  string  $method

     * @return void

**Comment by shifa7474** [comment]
how to resolve this issue

**Comment by erikn69** [comment]
You are reallly lost, that method is not from this package
